### PR TITLE
Changed MCServer Tracker Motd Path

### PR DESCRIPTION
**Pull request by DarkZo0m**
Last Motd Path Doesn't Support Some Emoji / Spacelines !

### CODE REVIEW

File: modules/tracker/mcserver.py [hunk 1]
```diff
@@ -85,9 +85,9 @@ def get_favicon_path(self):
 
     def get_motd_path(self):
         if self.status:
-            data = "http://status.mclive.eu/{}/{}/25565/banner.png".format(self.get_name(), self.server_address)
+            data = "https://api.loohpjames.com/serverbanner.png?ip={}&name={}".format(self.server_address(), self.get_name)
```
Comment (Alijkaz) [inline modules/tracker/mcserver.py:88]: There's a possible error occuring in this line
self.server_address() should be changed to self.server_address
and self.get_name should be self.get_name()
please fix these problems and do a fresh commit

**Review by Alijkaz — CHANGES_REQUESTED** [review]
Please fix thhese issues

File: modules/tracker/mcserver.py (hunk 1 above)
Comment (DarkZo0m) [reply modules/tracker/mcserver.py]: Fixed !

File: modules/tracker/mcserver.py [hunk 2]
```diff
@@ -85,7 +85,7 @@ def get_favicon_path(self):
 
     def get_motd_path(self):
         if self.status:
-            data = "https://api.loohpjames.com/serverbanner.png?ip={}&name={}".format(self.server_address(), self.get_name)
+            data = "https://api.loohpjames.com/serverbanner.png?ip={}&name={}".format(self.server_address, self.get_name()
```
Comment (Alijkaz) [inline modules/tracker/mcserver.py:88]: There still would be errors with this line, line is not correctly closed!
`format(self.server_address, self.get_name()`
should be changed to
`format(self.server_address, self.get_name())`

**Review by Alijkaz — CHANGES_REQUESTED** [review]
Please fix the issues first

**Pull request merged**